galaxy_ng/app/dynaconf_hooks.py

```diff
@@ -279,3 +280,10 @@ def configure_feature_flags(settings: Dynaconf) -> Dict[str, Any]:
     data["GALAXY_FEATURE_FLAGS__collection_auto_sign"] = settings.get(
         "GALAXY_AUTO_SIGN_COLLECTIONS", False)
     return data
+
+
+def configure_pulp_ansible(settings: Dynaconf) -> Dict[str, Any]:
+    return {
+        "ANSIBLE_URL_NAMESPACE": "galaxy:api:v3:",
+        "ANSIBLE_DEFAULT_DISTRIBUTION_PATH": "published"
+    }
```

Comment:
The main reason to use dynaconf_hooks is to be able to add conditional settings, in this case you don't have any kind of conditional. can't this be just variables on `settings.py` or you need it to be set on the late stage of settings loading?

Reply:
The problem with this approach is that this is no more overridable via environment variables, as hooks runs after envvars are loaded the values set here are the final one.

You can at least do:

```suggestion
import os

def configure_pulp_ansible(settings: Dynaconf) -> Dict[str, Any]:
    return {
        "ANSIBLE_URL_NAMESPACE": os.getenv("PULP_ANSIBLE_URL_NAMESPACE", "galaxy:api:v3:"),
        "ANSIBLE_DEFAULT_DISTRIBUTION_PATH": os.getenv("PULP_ANSIBLE_DEFAULT_DISTRIBUTION_PATH", "published")
    }
```

Reply:
@rochacbruno `ANSIBLE_URL_NAMESPACE` can't really be set to anything else or it will break all of our APIs, so that doesn't bother me.

I would like to set these in settings.py, but the pulp ansible settings file takes prescendence over ours, and I couldn't think of any other way to easily override these.

Reply:
in that case  that is ok